modules/configuration/src/configuration.ts

```diff
@@ -62,8 +72,19 @@ function resolveSpecs(config: Config) {
             })
         }
     }
+    return Promise.all(promises)
+}
+function getSpecFromUrl(url: string) {
+    return new Promise((resolve, reject) => {
+        request.get({
+            uri: url,
+            resolveWithFullResponse: true,
+            simple: false
+        }).then((response: any) => {
+            resolve(response.body)
```

Comment:
I see no error handling!!! What if the URL is wrong or not reachable anymore? I want to have a nice validation error!

Reply:
done